context/app/static/js/components/detailPage/ProcessedData/hooks.ts

```diff
@@ -40,26 +40,26 @@ export function useSortedSearchHits(datasets: ReturnType<typeof useProcessedData
 }
 
 /**
- * Formats the processed datasets' pipelines and their counts for presentation.
- * @param datasets The processed datasets to count the pipelines of.
- * @returns Text for the pipelines label, and additional text for each pipeline and its count.
+ * Formats the processed datasets' analyses and their counts for presentation.
+ * @param datasets The processed datasets to count the analyses of.
+ * @returns Text for the analyses label, and additional text for each analysis and its count.
  */
-export function usePipelineCountsInfo(datasets: Pick<ProcessedDatasetInfo, 'pipeline'>[]) {
-  const pipelines = datasets.map((dataset) => dataset.pipeline);
-  const pipelineCounts = pipelines.reduce(
-    (acc, pipeline) => {
-      acc[pipeline] = (acc[pipeline] || 0) + 1;
+export function useAnalysesCountInfo(datasets: Pick<ProcessedDatasetInfo, 'pipeline' | 'assay_display_name'>[]) {
+  const analyses = datasets.map((dataset) => dataset.pipeline ?? dataset.assay_display_name);
+  const analysesCount = analyses.reduce(
+    (acc, analysis) => {
+      acc[analysis] = (acc[analysis] || 0) + 1;
       return acc;
     },
     {} as Record<string, number>,
   );
```

Comment:
You could bring the logic from the `map` into the `reduce` to avoid iterating twice. Not worth addressing by itself.